natural_sort.hpp

```diff
@@ -196,7 +196,7 @@ namespace natural
 			if(*current2==' ') flag_found_space2 = true;
 			
 
-			if( !isdigit(*current1 ) && !isdigit(*current2))
+			if( !isdigit(*current1 ) || !isdigit(*current2))
 			{
 				//If Both Are Non Digits do Normal Comparision
```

Comment:
// Normal comparision if any of character is non digit character

Reply:
:+1: